package.json

```diff
@@ -15,8 +15,8 @@
     "bunyan": "1.5.1",
     "cookie-parser": "1.3.5",
     "express": "4.19.2",
-    "express-body-schema": "github:Synpheros/express-body-schema#master",
-    "express-fileupload": "1.2.1",
+    "express-body-schema": "github:e-ucm/express-body-schema#master",
+    "express-fileupload": "1.5.0",
     "keycloak-public-key": "github:e-ucm/keycloak-public-key#master",
```

Comment:
@jujusb chase any reference like this (to a github repo project both ours or others) and add a specific commit reference (https://stackoverflow.com/a/18049685) so builds are reproducible and to avoid any issue.

Reply:
fix using new tag versions